bench/src/main/scala/DietAddBench.scala

```diff
@@ -0,0 +1,81 @@
+/**
+  * Created by anicolaspp on 2/18/16.
+  */
+
+package dogs
+package bench
+
+import dogs.Predef._
+import org.openjdk.jmh.annotations.{Benchmark, Scope, Setup, State}
+
+import scala.util.Random
+import scalaz.Diev
+
+@State(Scope.Benchmark)
+class DietAddBench {
+
+  implicit object scalazEnumInt extends scalaz.Enum[Int] {
```

Comment:
why define your own instead of using the one from scalaz?

https://github.com/scalaz/scalaz/blob/abbd0238972d71bd07724218d4f85341ab720389/core/src/main/scala/scalaz/std/AnyVal.scala#L234